### PR TITLE
Update tprint, add show_font

### PR DESCRIPTION
**Pull request by 99Kies**
#### What does this implement/fix? Explain your changes.

Update tprint, add show_font

``tprint('huctf', font='random', show_font=True)``

### CODE REVIEW

File: art/art.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def randart():
     return art("random")
 
 
-def tprint(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
+def tprint(text, font=DEFAULT_FONT, show_font=False, chr_ignore=True, decoration=None):
```
Comment (sepandhaghighi) [inline art/art.py:206]: 1. Add `show_font`  at the end of parameters (after `decoration`)
2. Update docstring

File: art/art.py [hunk 2]
```diff
@@ -464,7 +467,7 @@ def __word2art(word, font, chr_ignore, letters, next_word):
     return result
 
 
-def text2art(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
+def text2art(text, font=DEFAULT_FONT, show_font=False, chr_ignore=True, decoration=None):
```
Comment (sepandhaghighi) [inline art/art.py:470]: 1. Add `show_font`  at the end of parameters (after `decoration`)
2. Update docstring

File: art/art.py [hunk 3]
```diff
@@ -487,6 +490,8 @@ def text2art(text, font=DEFAULT_FONT, chr_ignore=True, decoration=None):
     font = font.lower()
     if font != "mix":
         font = indirect_font(font, text)
+        if show_font:
+            print("#font name : %s\n" % font)
```
Comment (sepandhaghighi) [inline art/art.py:494]: use `.format` instead of this method ;-)

Comment (99Kies) [reply art/art.py]: 🆗, I got it.

**Comment by sadrasabouri** [comment]
> #### Reference Issues/PRs
> #### What does this implement/fix? Explain your changes.
> 
> Update tprint, add show_font
> 
> `tprint('huctf', font='random', show_font=True)`
> #### Any other comments?

> #### Reference Issues/PRs
> #### What does this implement/fix? Explain your changes.
> 
> Update tprint, add show_font
> 
> `tprint('huctf', font='random', show_font=True)`
> #### Any other comments?

Thank for your effort.
Please change `set_default` function properly in order to pass CI tests.
You can check [this](https://github.com/sepandhaghighi/art/pull/111/files#diff-33cd96c41d50ba12fa746b59fb9defbfR525-R543) PR that may be useful.

**Comment by 99Kies** [comment]
```bash

$ git  pull https://github.com/sepandhaghighi/art 
From https://github.com/sepandhaghighi/art
* branch            HEAD       -> FETCH_HEAD                                    
Already up to date.
```

hey, guys, how can I resolve this file conflict ?

**Comment by sadrasabouri** [comment]
> ```shell
> $ git  pull https://github.com/sepandhaghighi/art 
> From https://github.com/sepandhaghighi/art
> * branch            HEAD       -> FETCH_HEAD                                    
> Already up to date.
> ```
> 
> hey, guys, how can I resolve this file conflict ?

Hi.
Thank you for your effort.
It may help you:
```
git remote add upstream https://sepandhaghighi/art
git fetch upstream dev
git checkout dev
git merge upstream/dev dev
```
Then you should fix conflicts.

**Comment by sepandhaghighi** [comment]
@99Kies Hi
This PR was inactive for a long time so I closed it.
If you are still interested in this feature please open another PR and also please take a look at our [Contributing Guideline](https://github.com/sepandhaghighi/art/blob/master/.github/CONTRIBUTING.md).

Best Regards
Sepand Haghighi

**Pull request closed without merging**